src/gwt/src/org/rstudio/studio/client/workbench/addins/AddinsCommandManager.java

```diff
@@ -72,7 +73,7 @@ public void onAddinsKeyBindingsChanged(AddinsKeyBindingsChangedEvent event)
                @Override
                public void onAddinRegistryUpdated(AddinRegistryUpdatedEvent event)
                {
-                  rAddins_ = event.getData();
+                  MainWindowObject.rAddins().set(event.getData());
```

Comment:
Will this result in the main window object getting its value set repeatedly (x number of satellites) when the event is fired?